warehouse/oidc/views.py

```diff
@@ -139,7 +140,9 @@ def mint_token_from_oidc(request: Request):
         )
 
     # Associate the given issuer claim with Warehouse's OIDCPublisherService.
+    print("unverified_issuer", unverified_issuer)
     service_name = OIDC_ISSUER_SERVICE_NAMES.get(unverified_issuer)
+    print("service_name", service_name)
```

Comment:
```suggestion
    service_name = OIDC_ISSUER_SERVICE_NAMES.get(unverified_issuer)
```

Reply:
Noooooooo!...Sorry Woodruff.  I should have double checked things before pinging for review

Reply:
All good!